percentage-fee-handler/src/lib.rs

```diff
@@ -0,0 +1,263 @@
+// The Licensed Work is (c) 2022 Sygma
+// SPDX-License-Identifier: LGPL-3.0-only
+
+#![cfg_attr(not(feature = "std"), no_std)]
+
+pub use self::pallet::*;
+
+#[cfg(feature = "runtime-benchmarks")]
+mod benchmarking;
+pub mod weights;
+pub use weights::*;
+
+#[cfg(test)]
+mod mock;
+
+#[allow(unused_variables)]
+#[allow(clippy::large_enum_variant)]
+#[frame_support::pallet]
+pub mod pallet {
+	use frame_support::{dispatch::DispatchResult, pallet_prelude::*, traits::StorageVersion};
+	use frame_system::pallet_prelude::*;
+	use sp_std::boxed::Box;
+	use sygma_traits::{DomainID, FeeHandler};
+	use xcm::latest::{AssetId, Fungibility::Fungible, MultiAsset};
+
+	const STORAGE_VERSION: StorageVersion = StorageVersion::new(0);
+
+	/// Mapping fungible asset id with domain id to fee rate
+	#[pallet::storage]
+	pub type AssetFeeRate<T: Config> = StorageMap<_, Twox64Concat, (DomainID, AssetId), u32>;
```

Comment:
Substrate has provided some useful percentage types, we can consider using [Permill](https://paritytech.github.io/substrate/master/sp_arithmetic/per_things/struct.Permill.html) in our scenario

Reply:
we actually are thinking to us BPS instead of a pure number range or a pure percentage here